source/leuven/tier2_hardware/wice_hardware.rst

```diff
@@ -44,3 +44,41 @@ The nodes are connected using an Infiniband HDR-100 network, the islands are ind
 
 .. figure:: wice_hardware/wice.png
    :alt: wICE hardware diagram
+
+
+Hardware details (extension)
+----------------------------
+We are currently installing and testing additional hardware for wICE,
+which will be made accessible during the spring of 2024:
+
+- 68 thin nodes
+
+  - 2 Intel Xeon Platinum 8468 CPUs (Sapphire Rapids),
+    48 cores each :raw-html:`<br />`
+    The base and max CPU frequencies are 2.1 GHz and 3.8 GHz, respectively.
+  - 256 GB RAM
+  - 960 GB SSD local disk
```

Comment:
I think we expose only a portion of this to the user, but of course you can argue that these are the correct hardware specs and a limit for users belongs rather in the quick start guide (also see our internal note https://scm.icts.kuleuven.be/issues/83697#note-8).

Reply:
We indeed consider the hardware detail pages to just be the 'raw' specs.